htdocs/adherents/ldap.php

```diff
@@ -209,7 +209,6 @@
 	}
 
 	$ldap->unbind();
-	$ldap->close();
```

Comment:
Why removing the ldap->close() ?

Reply:
@eldy this is an alias of unbind in php doc... Same function.